spec/ruby/core/enumerable/min_spec.rb

```diff
@@ -88,4 +86,21 @@
     multi.min.should == [1, 2]
   end
 
+  context "when called with an argument n" do
+    context "without a block" do
+      it "returns an array containing the minimum n elements" do
+        result = @e_ints.min(2)
+        result.should == [22, 333]
+        result.should be_an_instance_of(Array)
```

Comment:
This seems a little bit redundant since the above line already compares `result` with an Array.